src/test/resources/com/digitalasset/testing/cucumber.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Cucumber support in the Functional Testing library
+  Cucumber support needs to work.
+
+  Background:
+    Given Sandbox is started with DAR "src/test/resources/ping-pong.dar"
```

Comment:
What are those party names below? The sentence should suggest. e.g. `Sandbox is started with DAR  ".." and the following parties` or similar

Reply:
:+1: